### DISCUSSION TITLE
Github Pages Hosting Problem

### DISCUSSION THREAD

**Discussion by Aheli-D** [community/community#62181] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My site is ready for testing, so I am publishing the repository. I changed my domain to a custom one, and after some bugs and things, I made it work. Then I found a bug and after I fixed it, I went back and decided I didn't want a custom domain. I clicked remove. So I tried to visit my site but it said "Page not found". Under that, it also showed my old custom domain. I went back and deleted the text in my CNAME file but it still didn't work. This is my first time using Github and I don't know what to do. Please help!

**Comment by radar-blip** [comment]
Try deleting the CNAME file like below:

![deletio](https://github.com/community/community/assets/39196767/ad94cb93-c940-4470-8623-8624bc09092b)

Also, the link to your `style.css` needs to be between the head tags, not the body tags.
This one: `<link rel="stylesheet" type="text/css" href="style.css">`

**Reply by Aheli-D** [reply]
Thank you for your feedback. I will fix it fight now!

**Comment by Aheli-D** [comment]
I deleted the CNAME but the same thing is happening.
![Screenshot 2023-07-31 2 59 55 PM](https://github.com/community/community/assets/140858201/3e469353-99e0-4118-bf9c-b30f37999b8f)
